bridge_adaptivity/module/models.py

```diff
@@ -52,11 +52,17 @@ def _load_cls_from_applicable_module(module_path, mod_name, class_startswith=Non
 }
 GRADING_POLICY_CHOICES = ((k, v.public_name) for k, v in GRADING_POLICY_NAME_TO_CLS.items())
 
+PROBLEM_ACTIVITY_TYPES = [
```

Comment:
I would use tuple here instead of a list.

Reply:
should I move this tuple to config/settings/base.py or somewhere in settings ? I think it should be there....